### PR TITLE
add github actions builds

### PR DESCRIPTION
**Pull request by gdams**
Adds a series of GH actions build checks for macOS and Linux x64

Screenshot 2020-09-15 at 14 03 02

### CODE REVIEW

**Comment by smlambert** [comment]
related: https://github.com/AdoptOpenJDK/build-jdk

FYI @sophia-guo

**Comment by gdams** [comment]
can be seen to be working in https://github.com/gdams/openjdk-build/pull/2

**Comment by aahlenst** [comment]
Looks great. What about Windows? What's the objective of the PR? Making some PR testing builds on our infra obsolete, getting more coverage?

**Comment by gdams** [comment]
> Looks great. What about Windows? What's the objective of the PR? Making some PR testing builds on our infra obsolete, getting more coverage?

Windows will come next. I need to work out the build environment first. The objective is to allow us to rely less heavily on the already heavily overloaded PR tester. Most build script changes can be tested with github actions.

File: sbin/build.sh [hunk 1]
```diff
@@ -707,8 +707,12 @@ moveFreetypeLib() {
 
   # codesign freetype before it is bundled
   if [ ! -z "${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}" ]; then
-    ENTITLEMENTS="$WORKSPACE/entitlements.plist"
-    codesign --entitlements "$ENTITLEMENTS" --options runtime --timestamp --sign "${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}" "${SOURCE_LIB_NAME}"
+    if touch test && codesign --sign "Developer ID Application: London Jamocha Community CIC" test && rm -rf test; then
```
Comment (karianna) [inline sbin/build.sh:710]: Worth a comment saying what this effectively does.

**Review by karianna — CHANGES_REQUESTED** [review]

File: sbin/build.sh (hunk 1 above)
Comment (gdams) [reply sbin/build.sh:711]: added

**Review by Willsparker — APPROVED** [review]
LGTM

**Pull request merged**